eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState/IncrementalStake.hs

```diff
@@ -200,8 +209,13 @@ incrementalStakeDistr incstake ds ps =
     UMap tripmap ptrmap = dsUnified ds
     PState {psStakePoolParams = poolParams} = ps
     delegs_ = UM.viewToVMap (delegations ds)
-    -- A credential is active, only if it is being delegated
-    step1 = resolveActiveIncrementalPtrs (`VMap.member` delegs_) ptrmap incstake
+    -- A credential is active, only if it is being delegated: (dom activeDelegs ◁ credStake)
+    activeCreds = Map.filterWithKey (\k _ -> VMap.member k delegs_) credStake
+    ignorePtrs = HardForks.forgoPointerAddressResolution (pp ^. ppProtocolVersionL)
+    step1 =
+      if ignorePtrs
+        then activeCreds
+        else resolveActiveIncrementalPtrs (`VMap.member` delegs_) ptrmap (IStake activeCreds ptrStake)
```

Comment:
The `step1A` was extracted from `resolveActiveIncrementalPtrs`, so the function looses its usefulness in my opinion and makes it just a little more confusing. I think it'll be cleaner if we inline it:
```suggestion
        else Map.foldlWithKey' addResolvedPointer activeCreds ptrStake
    addResolvedPointer ans ptr coin =
      case Map.lookup ptr ptrsMap of -- Map of ptrs to Credentials
        Just cred
          | VMap.member cred delegs_ -> Map.insertWith (<>) cred coin ans
        _ -> ans
```

Could you also rename `ptrmap` to `ptrsMap` and `tripmap` to `triplesMap`?

Reply:
Love this version, less noise! 
I suppose we could look up directly in `activeCreds` instead, since we need that map anyway for the "forgo" case, so
`Just cred | Map.member cred activeCreds ->  Map.insertWith (<>) cred coin ans`

Reply:
> I suppose we could look up directly in activeCreds instead

I am not sure we can do that. I don't think this is equivalent: `Map.member cred activeCreds` === `VMap.member cred delegs_` for `cred` in the PtrsMap. Because it could be the case that cred is in`deleg_`, but not in `activeCreds`, because it is not part of `credStake`, but instead is part of `ptrStake`